foundation-extensions/mobile-core/configuration/README.md

```diff
@@ -263,7 +263,41 @@ ACPCore.UpdateConfiguration(config);
 {% endtab %}
 {% endtabs %}
 
-## Clearing programmatic updates to the configuration (iOS 3.x.x Only)
+## Clearing programmatic updates to the configuration (Android and iOS 3.x.x Only)
```

Comment:
can we use the iOS (AEP 3.x) notation in these documents to be uniform with the other docs?